README.md

```diff
@@ -1122,6 +1122,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Justine Syde Bulacito](https://github.com/celesica)
   - [Jyoti KM](https://github.com/JyotiKM29)
   - [Jyotiraditya](https://github.com/Jyotiraditya24)
+  - [Joshua Olorunnipa](https://github.com/JoshuaRotimi)
```

Comment:
Please add your name in line 1110, after Joshua Nweze.
Let us know if you need any help with that. 🙂